website/pages/blog/postgres-cdc-to-any-destination.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: PostgreSQL CDC (Change-Data-Capture) to any destination
+tag: announcement
+date: 2023/12/14
+description: >-
+  In this blog we will talk about how to stream changes from PostgreSQL
+  to any destination with CloudQuery and no queue or Kafka required.
+author: yevgenypats
+---
+
+import { BlogHeader } from "../../components/BlogHeader"
+
+<BlogHeader/>
+
+One of the most powerful features of PostgreSQL is its ability to replicate data, and in this article, we'll take a deep dive into the topic of logical replication.
+
+## What is CDC?
+
+[CDC (Change-Data-Capture)](https://en.wikipedia.org/wiki/Change_data_capture) is a term used in a set of design patterns in databases where one can track changes to the data and act upon it.
+In PostgreSQL, CDC can be achieved via a feature called logical replication.
```

Comment:
```suggestion
In PostgreSQL, CDC can be achieved using the [logical replication](https://www.postgresql.org/docs/current/logical-replication.html) feature.
```
Could also refer to a specific PG version rather than `current`